libr/include/r_util/r_sys.h

```diff
@@ -26,6 +26,7 @@ R_API void r_sys_set_environ(char **e);
 R_API ut64 r_sys_now(void);
 R_API const char *r_time_to_string (ut64 ts);
 R_API int r_sys_fork(void);
+R_API void r_sys_exit_immediately(int status); // _exit(int status)
```

Comment:
what about r_sys_exit() instead?

Reply:
That would suggest it to be regular `exit()`, not `_exit()`.